ReleaseNotes.md

```diff
@@ -0,0 +1,82 @@
+`JustSaying`
+`JustSaying.Models`
+
+Each containing just 1 assembly.
+`JustSaying.Models` actually already exists, at version `2.0.0.x`, but then disappeared from the solution after.
+If you are using `JustSaying` v4 and any version of `JustSaying.Models`, then you could run into runtime issues, you will likely need to upgrade.
+
+### More async
+We now have a `PublishAsync` method for example. See known issues for where we are not yet async.
+
+### Logging abstracted
+In v4 of `JustSaying` and below, `NLog` was a hard dependency, this wasn't very friendly to people that don't use `NLog`, so we have abstracted this.
+We setting on using `Microsoft.Extensions.Logging.Abstractions` rathen than reimplementing the wheel.
+There are [many providers](https://github.com/aspnet/Logging#providers) today you can use, including `NLog`, `Serilog` and Microsoft provided ones such as Console logger etc...
+
+Logging is a manditory requirement for using `JustSaying`, if you want to turn off logging you should start your fluent configuration with:
+```csharp
+.WithLogging(new LoggerFactory())
+```
+
+### Message handleing backoff strategies
+
+### Ability to handle publish exceptions per publisher
+You can now configure publishers and set the `HandleException` property with a delegate that can deal with the exception, and return if it has been handled or if it wants the exception to propogate.
+
+### Embedded source and embedded pdbs
+The assembies now have the pdbs embedded, and to make debugging completely dependency free, the pdbs have the source embedded. Hurray!
+
+### All the fixes cherry picked from v4
+As this was such a big change, v5 was developed along side v4, which contained some smaller (but important) changes, if you are upgrading from 4.0 then you will get the benifit from these too:
+
+
+### Discovering Topics more efficiently
+When checking to see if an SNS Topic already exists, we were using the FindTopic AWSSDK method, which you might expect to make a single API call to return a single SNS Topic. Unfortunately it doesn't do this, within this method it calls ListTopics, then it picks out the topic in memory. For AWS accounts with a large number of topics, this became very slow.
+
+We now don't make this check, as creating a topic is idempotent, this results in a rapid startup time for users under the above conditions.
+
+## Upgrade instructions
+
+
+## Known issues
+
+### Synchronous initialization.
+The fluent configuration is not lazy, and does asynchronous work (on non-full .NET AWSSDK is async only), but does so in a blocking way. The configuration is designed to be run once in your application in a context where blocking and thead usage should not be an issue.
+
+We are aware that this isn't ideal, and have a branch where we are fixing it by adding an awaitable `.BuildAsync()` method, this is a reasonable breaking change so we want to take the opertunity to refactor and rationalise the fluent api design, this will likely come in v6.
```

Comment:
Typo _opertunity_

Reply:
I'd also say "...rationalise the fluent api design which will likely come in v6."